packages/web-server/src/bin.ts

```diff
@@ -0,0 +1,17 @@
+import { hideBin } from 'yargs/helpers'
+import yargs from 'yargs/yargs'
+
+import { bin } from '../package.json'
+
+import { description, builder } from './cliConfig'
+import { handler } from './cliConfigHandler'
+
+// Use the name of the bin as `scriptName` for Yargs to keep things in sync.
+// Obviously the way it's written it depends on there being only one bin
```

Comment:
Add a unit test to make sure? And/or throw an error here?

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9884/commits/9cb42cc158c249e4fd29fd65cd09d7a484fe4518